src/main/scala/no/ndla/searchapi/model/search/EmbedValues.scala

```diff
@@ -8,7 +8,7 @@
 package no.ndla.searchapi.model.search
 
 case class EmbedValues(
-    id: Option[String],
+    id: List[String],
```

Comment:
Vil ikkje dette knekke indeksen igjen? Er det bedre å la id ligge og så legge til ids for å unngå det?

Reply:
Eller du endrer kanskje ikkje på mappinga her?

Reply:
Om navnet ikke endres skal det fungere fint. Elasticsearch ser ut til å håndtere både strenger og lister av strenger med samme indeksdefinisjon. 

Er enig i at `ids` er et bedre navn, men om det endres tror jeg søket brekker fram til det er indeksert på nytt.

Reply:
Det var knekking av indeks eg var redd for. Men så lenge du ikkje endrer navn så skal nok det gå bra.